pkg/docker/events.go

```diff
@@ -170,3 +179,14 @@ func createPortMapping(ports []types.Port) (nat.PortMap, error) {
 
 	return portMap, nil
 }
+
+// Removes entries in port mapping that do no hold any values
+// for IP and Port e.g 9000/tcp:[].
+func validatePortMapping(portMap nat.PortMap) {
```

Comment:
Hmm, I think returning a new `nat.PortMap` with just the acceptable ports would be better, but it doesn't really matter here.

Reply:
I agree, but I was concerned about extra allocation if this thing gets hit hard.